src/seedu/addressbook/commands/Command.java

```diff
@@ -39,9 +39,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
+    /*public CommandResult execute(){
```

Comment:
avoid keeping unused code around. if you think it might still be useful, dont worry, that is what git is for so just go ahead and remove it